scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -55,7 +55,7 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                                 <button class="sd-checkbox sd-checkbox--button-style"
                                     ng-class="{checked: personalShowSent}"
                                     ng-click="togglePersonalShowSent()"
-                                    translate>Sent Items</button>
+                                    translate>{{ gettext('Sent Items') }}</button>
```

Comment:
this should work via `translate` directive, is it not being extracted or translated?

Reply:
It was not on the `.po` file 🤔

Reply:
well it might not be translated on transifex, so better check if it's in pot file after extraction

Reply:
a translate directive should be enough for it to be extracted. I was testing the translations and the ones from `gettext` calls are only extracted from js/ts files, not .html. We shouldn't be using gettext in `.html` files. If it's not extracting it from a HTML file there's likely some parsing error, try removing sections of the file until it gets extracted.

Reply:
Yeah I updated it and generated the corresponding `.pot`